src/routes/user/validation.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('@hapi/joi');
+
+export const registerValidation = (data) => {
+  const schemaUser = Joi.object({
+    name: Joi.string().pattern(new RegExp('^[A-Za-z0-9]{6,255}$')).required(),
```

Comment:
At least some special characters should probably be allowed here for instance to make name patters like `Name Surname-Surname` common in Poland possible. Also, having separate validations for the min and max characters will give more customized error messages.

Reply:
I gave up the name pattern and gave only the minimum number of characters. It can be like that?